docs/resources/ocean_aws_extended_resource_definition.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: ocean_aws_extended_resource_definition"
+subcategory: "Ocean"
+description: |-
+  Provides a Spotinst Extended Resource Definition resource.
```

Comment:
cahnge to   "Provides a Spotinst Extended Resource Definition for Ocean AWS cluster resource."
(need to approve with product team)